test/helpers/governance.js

```diff
@@ -1,3 +1,4 @@
+const { web3 } = require('hardhat');
```

Comment:
Do we need that to be explicit ?

Reply:
There are many place where we use implicit import for things like `web3`, and a few places where we have explicit import. IMO we need some guideline (or we need to move the ethers.js)

Reply:
> or we need to move the ethers.js

This doesn't change anything, as long as we use Hardhat we have to decide whether to use globals or not.

Personally I always prefer explicit imports.